src/modules/import/domain/import-event-handler.ts

```diff
@@ -18,12 +18,16 @@ import { OrganisationByIdAndName } from './import-workflow.js';
 import { Injectable } from '@nestjs/common';
 import { ClassLogger } from '../../../core/logging/class-logger.js';
 import { ImportPasswordEncryptor } from './import-password-encryptor.js';
+import { PersonPermissions } from '../../authentication/domain/person-permissions.js';
+import { ImportDomainError } from './import-domain.error.js';
 @Injectable()
 export class ImportEventHandler {
     public selectedOrganisationId!: string;
 
     public selectedRolleId!: string;
 
+    private readonly NUMBER_OF_USERS_PRO_REQUEST: number = 25;
```

Comment:
Name is a bit misleading, how about `USER_IMPORT_BATCH_SIZE` to signify, that this is used for batching?